### PR TITLE
fix(#210): Reduce chances of XSS injections

### LINKED ISSUE #210: xss injection
### Environment
* python version 3.8.4
* django Version 3.2.2
* django-comments-dab 2.6

### Describe the bug
your form has no protection against xss injections
Sorry in advance, but I checked it out on your live demo
don't worry i deleted my comment

### Steps To Reproduce
1. publish this code in form for comment
<script>alert('Test alert');</script>

### Screenshots
image

### PR DESCRIPTION
**Pull request by abhiabhi94**
- on the content field.

fixes #210

### CODE REVIEW

File: comment/templates/comment/comments/content.html [hunk 1]
```diff
@@ -4,10 +4,10 @@
     {% include "comment/comments/urlhash.html" with urlhash=urlhash %}
     {% endblock comment_urlhash %}
     {% if text_2 %}
-        {{ text_1|safe }}
```
Comment (Radi85) [inline comment/templates/comment/comments/content.html:7]: I believe this will break this #169

**Review by Radi85 — CHANGES_REQUESTED** [review]

File: comment/templates/comment/comments/content.html (hunk 1 above)
Comment (abhiabhi94) [reply comment/templates/comment/comments/content.html:7]: yes, you are right. I have made changes to the `render_content` template tag to first escape the original content and then marked it as safe after our change.

**Review by Radi85 — DISMISSED** [review]

**Comment by Radi85** [comment]
Great work!
@abhiabhi94 Could you please rebase your PR?

**Comment by abhiabhi94** [comment]
Hey @Radi85, this status reporting in taking too long on travis-ci. It maybe due to an issue at either end. Do you consider switching to GitHub actions? A lot of open-source repositories have migrated to it, and I too have had satisfactory experience.

Although, It maybe a backroom strategy by Microsoft to have this migration.

**Comment by Radi85** [comment]
> Hey @Radi85, this status reporting in taking too long on travis-ci. It maybe due to an issue at either end. Do you consider switching to GitHub actions? A lot of open-source repositories have migrated to it, and I too have had satisfactory experience.

I have no never used actions but I would not mind to migrate since we are facing issues with Travis. I will have a look at it as soon as I have a time.

> Although, It maybe a backroom strategy by Microsoft to have this migration.

Microsoft 😆

**Review by Radi85 — APPROVED** [review]

**Comment by abhiabhi94** [comment]
> I have no never used actions but I would not mind to migrate since we are facing issues with Travis. I will have a look at it as soon as I have a time.

I will try to make a patch for this one soon.

**Comment by abhiabhi94** [comment]
One more thing, you'll have to add the password token to the secrets for automated release to PyPI with actions. I would suggest using a manual release as there have been instances in the past when secrets have been leaked(also I don't like microsoft).

I can handle the release process, if you don't have the time. In case, you have any other suggestions, they are welcome.

**Pull request merged**